app/assets/javascripts/discourse/app/components/d-editor.gjs

```diff
@@ -162,38 +161,6 @@ export default class DEditor extends Component {
     return keymap;
   }
 
-  @action
```

Comment:
Why did we remove all this code @jdmartinez1062 ?

Reply:
ok I see, this is a msitkae of your other PR where you moved this and forgot to remove it :+1: